recipes/openimageio/3.x/test_package/conanfile.py

```diff
@@ -0,0 +1,25 @@
+from conan import ConanFile
+from conan.tools.cmake import cmake_layout, CMake
+from conan.tools.build import can_run
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
+    test_type = "explicit"
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
+    def layout(self):
+        cmake_layout(self)
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not can_run(self):
```

Comment:
```suggestion
        if can_run(self):
```

This bug is also present in `all` folder.